docs/client/install-kn.md

```diff
@@ -27,6 +36,59 @@ Links to the latest nightly-built executable binaries are available here:
 - [Linux](https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64){target=_blank}
 - [Windows](https://storage.googleapis.com/knative-nightly/client/latest/kn-windows-amd64.exe){target=_blank}
 
-## Using kn with Tekton
+### Using kn with Tekton
 
 See the [Tekton documentation](http://hub.tekton.dev/tekton/task/kn){target=_blank}.
+
+## Install the Knative Operator CLI Plugin
```

Comment:
I'm okay with leaving a note here (though it seems odd that we mention the administrator-focused operator plugin but not the developer-focused `func` plugin), but this section should make it clear that the operator subcommands are mostly useful for cluster administrators, rather than average developers.